### DISCUSSION TITLE
Script feedback

### DISCUSSION THREAD

**Discussion by Anomander43** [gordonbay/Windows-On-Reins#11] [Q&A]
Hi,

I just tried this script and mostly understand how to set and control it, but have couple of questions so far.

I noticed that for the "doFingerprintPrevention" we have basically the same value for one of the items:
RegChange "SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\Explorer" "NoRecycleFiles" "1"
Only the description differs.

Also I have some network problems after running the script with mostly using the recommended values.
An example would be the Hades game can't sync to the cloud.
Or the viber application can't send videos. Texts go through although much slower than before.
I also just installed Rogue Legacy on Epic Store, but the installation failed, kinda. The launcher says it's installed just fine, but the actual files of the game are nowhere to be found.
I'm guessing all of the above problems are some network disabled/protection thingy, but I'm not sure where to look.
Do you may be have any suggestion that I could try? Thanks!

**Comment by gordonbay** [comment]
just reverse the changes one by one, doQualityOfLifeStuff, doPerformanceStuff... and track what is causing the problem

**Comment by ghost** [comment]
Good day how can I revert changes? I can't logon on to my Microsoft office 365 anymore.
